components/Auction-Schedule.jsx

```diff
@@ -0,0 +1,260 @@
+import React, { useEffect, useState } from 'react';
+import { ApiPromise, WsProvider } from "@polkadot/api";
+import { 
+	PolkadotAuctions,
+	PolkadotSlotLeasePeriod,
+	PolkadotSlotLeaseOffset,
+	PolkadotLeasePeriodPerSlot,
+	KusamaAuctions,
+	KusamaSlotLeasePeriod,
+	KusamaSlotLeaseOffset,
+	KusamaLeasePeriodPerSlot
+} from './utilities/auctionVariables';
+
+let API = undefined;
+let ChainState =  {
+	block: undefined,
+	blockNumber: undefined,
+	blockDate: undefined,
+}
+let Options = [];
+const FutureBlock = "0x0000000000000000000000000000000000000000000000000000000000000000";
+
+// Component for displaying auction data
+function AuctionSchedule() {
+	const [auctions, setAuctions] = useState("Loading Auctions...");
+
+	useEffect(async () => {
+		const title = document.title;
+		if (title === "Parachain Slot Auctions · Polkadot Wiki") {
+			const chain = "Polkadot";
+			const wsProvider = new WsProvider("wss://rpc.polkadot.io");
+			await LoadOptions(PolkadotAuctions, wsProvider);
+			await LoadBlockCacheThenUpdate(chain, PolkadotAuctions, setAuctions, { target: { value: 0 } });
+		}
+		else if (title === "Parachain Slot Auctions · Guide") {
+			const chain = "Kusama";
+			const wsProvider = new WsProvider("wss://kusama-rpc.polkadot.io");
+			await LoadOptions(KusamaAuctions, wsProvider);
+			await LoadBlockCacheThenUpdate(chain, KusamaAuctions, setAuctions, { target: { value: 0 } });
+		}
+		else {
+			console.log("Unknown wiki/guide type");
+		}
+	}, []);
+
+	// Render
+	if (auctions !== undefined) {
+		return auctions;
+	} else {
+		return (<div>Loading auction data...</div>)
+	}
+}
+
+// Loads drop-down selections
+async function LoadOptions(auctions, wsProvider) {
+	for (let i = 0; i < auctions.length; i++) {
+		const option = <option value={i} key={i}>{`Auction #${i + 1} at Block #${auctions[i]["startBlock"]}`}</option>
```

Comment:
As we display the block numbers in the content. Makes it look less cluttered

```suggestion
		const option = <option value={i} key={i}>{`Auction #${i + 1}`}</option>
```